README.md

```diff
@@ -29,7 +29,7 @@
 
 ## References
 *References for additional information about the Kinesis keyboard*
--   [Advantage2-SmartSet-App](https://github.com/KinesisCorporation/Advantage2-SmartSet-App) - Open sourced GUI for modifying the Advantage 2
+-   [Advantage2-SmartSet-App](https://github.com/KinesisCorporation/SmartSetApps) - Open sourced GUI for modifying the Advantage 2
```

Comment:
Thank you very much for providing the updated link. Let's update the link text and the description to match the new repository. 

```suggestion
-   [KinesisCorporation/SmartSetApps](https://github.com/KinesisCorporation/SmartSetApps) - SmartSet apps for keyboards, foots pedals and more 
```